README.md

```diff
@@ -1,97 +1,99 @@
-/medstore/Development/WGS_validation/in_silico_panels/
+### The genes/transcripts can’t be found
+Sometimes genes or transcripts can not be found in `refseq_20190301_ncbiRefSeq`. It could be that the transcript has been updated since 2019-03-01 or that the gene name has been changed. If an element of the input list was not found, the script will output a file named notfound.txt. Should the gene/transcript not be present in `refseq_20190301_ncbiRefSeq`there are a few things you can try to solve it:
 
-Men nu tänker jag att vi ska spara bedfilerna i det här repot, eftersom filerna ändå är ganska små och det möjliggör versionshantering! FANTASTISKT JU. 
+**Transcripts**
+Try deleting the version part of the transcript name in case there is another version present. If there is, check with the person requesting the panel update if it is okay to use the other version.
 
-För att vara lite bättre organiserad har jag sparat alla genlistor/transkriptlistor som har skickats i detta googlesheet. "InSilicopaneler KK" & "InSilicopaneler KG".
+**Genes names**
+Check if the gene has another name (e.g. by using [GeneCards database](https://www.genecards.org) ) that might be present in `refseq_20190301_ncbiRefSeq`. To replace a name use the script `replace_names.sh <old name> <new name>`. The changes will be saved in `changes_made`.  
+<br />
 
-# coverage analysis (for in silico panel verification into WGS analysis) 
+## The insilico\_panels folder
 
-you need to be root to run
+The in silico panels used in wopr are also stored here. Every panel folder contains a gene or transcript list as well as a bed file created by create\_bed.py. The lists and bed files of the panels currently used by wopr should also be in `WOPR/references/insilico_panels/`. 
+<br />
 
-cd to validate_wrapper subdirectory
+## Coverage analysis (for in silico panel verification into WGS analysis)
 
+In order to create a coverage analysis you will need a bed file for the panel in question.
+
+### How to run
+```
+# run as root
+cd validate_wrapper/
 ./validate_wrapper /path/to/bedfile.bed [departmentID = KG/KK,PAT] [annotationlevel = 1/2/3/4/5]
 
-results are saved here:
+# example:
+./validate_wrapper.sh /path/to/insilico_wgs/insilico_panels/KK/rubbningar_i_kopparmetabolism.v1.0/rubbningar_i_kopparmetabolism.v1.0.bed KK 1
+```
+### Result directory
+```
 /medstore/Development/WGS_validation/in_silico_panels/${department}/validate/panels/
+```
+
+## How to get the panel validated
+Some of the results from the coverage analysis are to be sent to the responsible clinician as an excel file. There is a template excel that you should fill in as described below.
 
-Now you need to extract various stats and add to an excelfile to be sent to responsible clinician for the insilico coverage verification (its a bit silly, it always works, and we should probably take up the fight on this at some point to save time for ourselves)
+**Template excel:** boxsync - /Dokument/WGS/konstitutionell/insilico
```

Comment:
How can "boxsync" be accessed?

Reply:
My guess this is some kind of remnant from when we used Box at GU (dropbox like service). Instead we should refer to the sharepoint